frontend/src/pages/order/libs/components/libs/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { type OrderCreateRequestDto } from '~/packages/orders/orders.js';
+
+const CREATE_ORDER_DEFAULT_PAYLOAD: OrderCreateRequestDto = {
+  customerName: '',
+  customerPhone: '',
+  scheduledTime: '',
+  startPoint: '',
+  endPoint: '',
+  carsQty: 1,
+  truckId: 0,
+};
```

Comment:
add TruckCarsQuantity enum in shared
```suggestion
const getCreateOrderDefaultPayload = (selectedTruckId: number): OrderCreateRequestDto => {
  return {
    customerName: '',
    customerPhone: '',
    scheduledTime: '',
    startPoint: '',
    endPoint: '',
    carsQty: TruckCarsQuantity.MIN,
    truckId: selectedTruckId
  }
}
```